src/plugins/plugins/permissions.js

```diff
@@ -62,13 +63,33 @@ export class Permissions extends ChatPlugin {
   @respond(/^permissions view user (\w+)$/i);
   async viewUser ([match, name], message) {
     await this.databaseInitialized();
+    let userIdDirty;
+    try {
+      userIdDirty = await this.bot.adapters[message.adapter].getUserIdByUserName(name);
+    } catch (err) {
+      this.bot.log.warn(err);
+    }
+    const userId = Permissions.nameToId(userIdDirty);
+    if (userId) {
+      const perms = this.bot.getRoles(userId);
+      return perms.join(', ');
+    }
+  }
 
-    const userId = Permissions.nameToId(
-      this.bot.adapters[message.adapter].getUserIdByUserName(name)
-    );
-
+  @help('/permissions view effetive user <user> to view all roles assigned to <user>');
+  @permissionGroup('role-management');
+  @respond(/^permissions view effective user (\w+)$/i);
```

Comment:
Thinking about it, here and elsewhere, we're using `(\w+)`, where you _could_ have a real-name username like "Jack Lawson" in some chat clients. We should just use `(.+)` instead, and then use convertToId to strip out non-\w characters